--- openmosixview-1.4/openmosixview/openmosixview.cpp.orig	Thu Apr 17 21:32:54 2003
+++ openmosixview-1.4/openmosixview/openmosixview.cpp	Thu Apr 17 21:32:30 2003
@@ -622,7 +622,7 @@
 	"\n" \
 	"The load is count in percent too so it just gives you an overview\n" \
 	"of the load. It does not exactly represent the the value of\n" \
-	"/proc/hpc/nodes/x/load of each node!\n" \
+	"/proc/mosix/nodes/x/load of each node!\n" \
 	"\n");
 
 	QLabel *e = new QLabel(tabs, 0, 0);
--- openmosixview-1.4/openmosixview/openmosixview.h.orig	Thu Apr 17 21:33:03 2003
+++ openmosixview-1.4/openmosixview/openmosixview.h	Thu Apr 17 21:33:18 2003
@@ -80,7 +80,7 @@
 
 
 #define clustercommand "mosctl"
-#define clusterdirectory "/proc/hpc/nodes/"
+#define clusterdirectory "/proc/mosix/nodes/"
 #define configfile "/root/.openMosixview"
 #define displayfile "/root/.openMosixview.disp"
 #define init1 "/etc/init.d/openmosix"
--- openmosixview-1.4/openmosixcollector/openmosixcollector/openmosixcollector.h.orig	Thu Apr 17 21:29:04 2003
+++ openmosixview-1.4/openmosixcollector/openmosixcollector/openmosixcollector.h	Thu Apr 17 21:29:22 2003
@@ -33,7 +33,7 @@
 
 
 #define collectorpid "/var/run/openMosixcollector.pid"
-#define clusterdirectory "/proc/hpc/nodes/"
+#define clusterdirectory "/proc/mosix/nodes/"
 #define logdirectory "/tmp/openmosixcollector"
 #define syncfile "sync.count"
 
--- openmosixview-1.4/openmosixmigmon/openmosixmigmon/openmosixmigmon.h.orig	Thu Apr 17 21:30:24 2003
+++ openmosixview-1.4/openmosixmigmon/openmosixmigmon/openmosixmigmon.h	Thu Apr 17 21:30:36 2003
@@ -54,7 +54,7 @@
 #include "omnode.h"
 
 #define clustercommand "mosctl"
-#define clusterdirectory "/proc/hpc/nodes/"
+#define clusterdirectory "/proc/mosix/nodes/"
 #define pi 3.1415926525
 #define eheight 600
 #define ewide 600
--- openmosixview-1.4/openmosixprocs/openmosixprocs/openmosixprocs.h.orig	Thu Apr 17 21:31:17 2003
+++ openmosixview-1.4/openmosixprocs/openmosixprocs/openmosixprocs.h	Thu Apr 17 21:31:57 2003
@@ -72,8 +72,8 @@
 
 
 #define clustercommand "mosctl"
-#define remotedir "/proc/hpc/remote/"
-#define clusterdirectory "/proc/hpc/nodes/"
+#define remotedir "/proc/mosix/remote/"
+#define clusterdirectory "/proc/mosix/nodes/"
 #define migratecommand "migrate"
 
 using namespace std;
